src/interceptors/access.interceptor.ts

```diff
@@ -4,13 +4,13 @@ import { ApiConfigService } from "src/common/api-config/api.config.service";
 import { HealthCheckController } from "src/endpoints/health-check/health.check.controller";
 
 @Injectable()
-export class JwtAuthenticateGuard implements CanActivate {
+export class JwtAuthenticateGlobalGuard implements CanActivate {
```

Comment:
Since this is a guard and not an interceptor, you can move it to the utils/guards folder and rename the file accordingly